### DISCUSSION TITLE
IntelliSense doesn't work anymore in VS code with nRF Connect extension

### DISCUSSION THREAD

**Discussion by Simon8890** [microsoft/vscode-discussions#497] [Extension Development QnA] [answered]
Hi, i have i problem with intellisense on vs code extention, nrf connect.
it's been tow days that intellisense doesn't work... how could i do to solve this problem?

**Comment by gjsjohnmurray** [comment, marked as answer]
Please use the Report Issue from VS Code's Help menu to raise this with the extension author.
